optimize/self-managed/optimize-deployment/configuration/system-configuration.md

```diff
@@ -292,20 +292,20 @@ Settings for automatic cleanup of historic process/decision instances based on t
-| historyCleanup.ingestedEventCleanup.enabled                                      | false         | A switch to activate the history cleanup of ingested event data. \[true/false\]                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                      |
+| YAML path                                                                        | Default value | Description                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
+| -------------------------------------------------------------------------------- | ------------- | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| historyCleanup.cronTrigger                                                       | `'0 1 * * *'` | Cron expression to schedule when the cleanup should be executed, defaults to 01:00 A.M. As the cleanup can cause considerable load on the underlying database it is recommended to schedule it outside of office hours. You can either use the default Cron (5 fields) or the Spring Cron (6 fields) expression format here. For details on the format please refer to: [Cron Expression Description](https://en.wikipedia.org/wiki/Cron) or [Spring Cron Expression Documentation](https://docs.spring.io/spring-framework/docs/5.1.0.RELEASE/javadoc-api/org/springframework/scheduling/support/CronSequenceGenerator.html) |
```

Comment:
I don't really like binding the link to a specific version of Spring given we update it to always be the latest. I also don't think these links are necessary really. Can we remove them entirely instead?

Reply:
Of course! To clarify, would you like to remove the links for both the expression description and the expression documentation?

Reply:
Yes, I think we can remove both links to keep it simple here 👍

Reply:
Adjusted in latest commit 👍